base/src/main/java/com/mindera/skeletoid/widgets/tooltip/PopupTooltip.kt

```diff
@@ -393,6 +336,11 @@ class PopupTooltip(
     private fun setupTooltipBehaviour(context: Context, layout: View, tooltipWidth: Int) {
         tooltip = PopupWindow(context)
 
+        //Handle timed dismiss
+        if (!indefinite) {
+            handler.postDelayed({ clearTooltip() }, showDuration)
+        }
+
         tooltip?.apply {
```

Comment:
Do we need this block? Can't we do it when instantiating the val

```
tooltip = PopupWindow(context).apply {
    height = ViewGroup.LayoutParams.WRAP_CONTENT
    width = tooltipWidth

    //Set the custom content view
    contentView = layout

    ...
}

if (!indefinite) handler.postDelayed({ clearTooltip() }, showDuration)
```

Reply:
sure 👍